runner/ansible/roles/2.1.3/tasks/main.yml

```diff
@@ -0,0 +1,39 @@
+---
+
+- name: Set Test ID
+  set_fact:
+    test_id: '2.1.3'
+
+- name: "{{ test_id }}.check"
+  shell: |
+    # SAP HANA Resource Agent is not configured, then pass
+    cibadmin -Q --xpath "//primitive[@type='SAPHana']/@type" || exit 0
+    # if configured and not msl, then fail
+    cibadmin -Q --xpath "//master/primitive[@type='SAPHana']/@type" || exit 1
+    # check the following values, if any do not match expected fail.
+    [[ $(cibadmin -Q --xpath "//primitive[@type='SAPHana']/operations/op [@name='monitor'] [@role='Master']" | grep -oP 'interval="\K[^"]+') != "{{ expected[test_id + '.master_interval'] }}" ]] && exit 1
```

Comment:
Could we put the `//primitive[@type='SAPHana']/operations/op` string in a variable? I see it repeated multiple times

Reply:
I think so, but I can't do that from where I am. I am fine you giving this a shot.

Reply:
Done